pkg/scheduler/logging.go

```diff
@@ -47,3 +47,14 @@ func logSnapshotIfVerbose(log logr.Logger, s *cache.Snapshot) {
 		s.Log(logV)
 	}
 }
+
+func getWorkloadReferences(targets []*preemption.Target) []klog.ObjectRef {
+	if len(targets) == 0 {
+		return nil
+	}
+	keys := make([]klog.ObjectRef, len(targets))
+	for i, t := range targets {
+		keys[i] = klog.KObj(t.WorkloadInfo.Obj)
+	}
+	return keys
```

Comment:
nit:
```suggestion
	return slices.Map(targets, func (t **preemption.Target) string (return klog.KObj((*t).WorkloadInfo.Obj)))
```
https://github.com/kubernetes-sigs/kueue/blob/a3688c4a1bd30743414c3e1bef71369dfde13521/pkg/util/slices/slices.go#L54C6-L54C9

Reply:
@trasc's has the same memory usage as the loop, and one less than the previous implementation with the intermediary slice `*workload.Info`. So this is good.